cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -277,21 +277,34 @@ func getSchedulerCommand(cfg *kubeadmapi.MasterConfiguration) []string {
 }
 
 // getProxyEnvVars builds a list of environment variables to use in the control plane containers in order to use the right proxy
-func getProxyEnvVars() []v1.EnvVar {
+func getProxyEnvVars(cfg *kubeadmapi.MasterConfiguration) []v1.EnvVar {
+	noProxyIPs := getNoProxyIPv4IPs(cfg)
```

Comment:
kubeadm must be able to handle both IPv4 and IPv6.

Reply:
@kad Thanks for reminding. This function works both for IPv4 and IPv6. Locally tested.